src/main/java/codeu/controller/ActivityfeedServlet.java

```diff
@@ -66,9 +66,14 @@ public void setMessageStore(MessageStore messageStore) {
 			activity.add(user);
 		}	
 		for (Conversation convo : conversations) {
+			User owner = userStore.getUser(convo.getOwnerId());
+			convo.setDisplayText(owner.getName() + " created conversation: " + convo.getTitle());
 			activity.add(convo);
 		}
 		for (Message message : messages) {
+			User author = userStore.getUser(message.getAuthorId());
+			Conversation conversation = conversationStore.getConversationWithID(message.getConversationId());
+			message.setDisplayText(author.getName() + " sent message: \"" + message.getContent() + "\"" + " to conversation: " + conversation.getTitle());
 			activity.add(message);
```

Comment:
I think this code looks great! It does what it needs to do and there is no error. I am little worried about the blackslash but I think it should be okay.

Reply:
Thanks! I used the backslashes in this because I wanted the messages on the activity feed page to show up in actual quotes, so I need those escape keys to get that.